shared/profile/container.js

```diff
@@ -183,20 +180,21 @@ const mergeProps = (stateProps, dispatchProps) => {
     onClearAddUserToTeamsResults: () => dispatchProps.onClearAddUserToTeamsResults(),
     onClickAvatar: () => dispatchProps._onClickAvatar(username),
     onClickShowcaseOffer: () => dispatchProps.onClickShowcaseOffer(),
-    onCopyStellarAddress: () => dispatchProps._copyStellarAddress(stateProps.stellarAddress),
+    onCopyStellarAddress: () => {
+      const maybeAddr = stateProps.trackerState.stellarFederationAddress
+      maybeAddr && dispatchProps._copyStellarAddress(maybeAddr)
+    },
     onFollow: () => dispatchProps._onFollow(username),
     onOpenPrivateFolder: () => {
       stateProps.myUsername && dispatchProps._onOpenPrivateFolder(stateProps.myUsername || '', username || '')
     },
     onRequestLumens: () => dispatchProps._onSendOrRequestLumens(username, true, 'keybaseUser'),
     onSearch: () => dispatchProps.onSearch(),
     onSendLumens: () => dispatchProps._onSendOrRequestLumens(username, false, 'keybaseUser'),
-    // TODO: shouldn't there be 'stellarFederatedAddress'?
     onSendOrRequestStellarAddress: (isRequest: boolean) =>
-      dispatchProps._onSendOrRequestLumens(stateProps.stellarAddress, isRequest, 'stellarPublicKey'),
+      dispatchProps._onSendOrRequestLumens(username, isRequest, 'keybaseUser'),
```

Comment:
big change is here where if you click on the address and then "Send" or "Request" from the menu that appears, it will try to send/request with username, not federation address (which would not have worked for requests anyway).